projects/sandbox/pyproject.toml

```diff
@@ -31,7 +31,7 @@ resnet = {layers = [3, 4, 6, 3], norm_groups = 16}
 repository_directory = "${BASE_DIR}/model_repo/" 
 force_generation = false
 Tb = 31536000  # 1 year of background
-inference_sampling_rate = 16
+inference_sampling_rate = 2
```

Comment:
Is this meant to still be in here since now we know that we don't need as frequent inference, or is this a leftover artifact from your studies?